docs/source/profiling.md

```diff
@@ -43,6 +43,12 @@ demo,hello,0,29.0,0.18
 These can then be parsed and plotted, as is done in the
 [experiment-microbench](https://github.com/faasm/experiment-microbench) repo.
 
+To run the same micro-benchmark in SGX, re-run the same command as follows:
+
+```bash
+WASM_VM="wamr" microbenchmark_runner <spec_file> <out_file> --sgx
```

Comment:
Re. our offline conversation about making SGX a VM type, can we do that and change this to:

```bash
WASM_VM="sgx" microbenchmark_runner <spec_file> <out_file>
```